pkg/runtime/runtime.go

```diff
@@ -36,13 +53,42 @@ type Info struct {
 	Trainer
 	// Scheduler parameters to add to the RuntimeJobTemplate.
 	*Scheduler
+	// TemplateSpec is TrainingRuntime Template object.
+	// ObjApply podSpecs and this PodSets should be kept in sync by info.SyncPodSetsToTemplateSpec().
+	TemplateSpec TemplateSpec
```

Comment:
So eventually, we will migrate Trainer fields to the PodSet ?

Reply:
Trainer fields will not be migrated in TrainJob. 
We will migrate the internal data structure `Info.Trainer` object to `Info.PodSet` as part of https://github.com/kubeflow/trainer/issues/2495.

Reply:
Yeah, sorry, I meant the `Info.Trainer`, as you said, so we can do it as part of #2495